src/main/java/com/uber/cadence/client/WorkflowClientOptions.java

```diff
@@ -72,6 +72,7 @@ private Builder(WorkflowClientOptions options) {
       metricsScope = options.getMetricsScope();
       identity = options.getIdentity();
       queryRejectCondition = options.getQueryRejectCondition();
+      contextPropagators = options.getContextPropagators();
```

Comment:
```suggestion
      contextPropagators = options.getContextPropagators();
      queryRejectCondition = options.getQueryRejectCondition();
```
extremely minor but might as well keep things in source order

Reply:
Thanks, done